src/pybind/mgr/dashboard/tox.ini

```diff
@@ -27,6 +27,8 @@ setenv =
     WEBTEST_INTERACTIVE = false
 commands =
     pytest {posargs}
+install_command =
+    pip install --find-links {toxinidir}/wheelhouse {opts} {packages}
```

Comment:
Doesn't `export PIP_FIND_LINKS="$TOX_INI_DIR/wheelhouse"` work? [[1]](https://pip.pypa.io/en/stable/user_guide/#environment-variables)

Reply:
@epuertat do you prefer export `PIP_FIND_LINKS` in run_tox.sh  ?

Reply:
That way you don't need to redefine install_command in any tox file. But I don't have any strong objection, so your call.

Reply:
@epuertat makes sense to me. updated and repushed.